src/rc/seedrng.c

```diff
@@ -404,8 +473,8 @@ int main(int argc _unused, char *argv[] _unused)
 	blake2s_update(&hash, &realtime, sizeof(realtime));
 	blake2s_update(&hash, &boottime, sizeof(boottime));
 
-	if (mkdir(SEED_DIR, 0700) < 0 && errno != EEXIST)
-		eerrorx("Unable to create \"%s\" directory: %s", SEED_DIR, strerror(errno));
+	if (mkdir(seed_dir, 0700) < 0 && errno != EEXIST)
+		eerrorx("Unable to create \"%s\" directory: %s", seed_dir, strerror(errno));
 
 	lock = open(LOCK_FILE, O_WRONLY | O_CREAT, 0000);
```

Comment:
You forgot to change `LOCK_FILE` here into `lock_file`, and so the code doesn't compile.